agent/config/config.go

```diff
@@ -29,6 +29,9 @@ type ListenerConfig struct {
 	// Addr is the address of the upstream service to forward to.
 	Addr string `json:"addr" yaml:"addr"`
 
+	// InsecureSkipVerify allows the proxy to ignore certificates on HTTPS connections.
+	InsecureSkipVerify bool `json:"insecure_skip_verify" yaml:"insecure_skip_verify"`
```

Comment:
Nice, I hadn't thought about TLS/HTTPS to the listeners upstream before.

Though thinking long term, this should probably be a separate `ListenerConfig.TLS` configuration, then it can also support `RootCAs`. This will also require supporting for TCP not just HTTP...

Reply:
Updated the format to the format you suggested in 80b355cb00ed4d19efee01f53ec9513639e3d829.

I have not yet changed anything in the TCP configuration, I will look into it.